cmd/manager/cmd.go

```diff
@@ -175,7 +176,10 @@ func NewCmd(scheme *runtime.Scheme) *cobra.Command {
 					handlers.Freeze{},
 				},
 				routes.TenantControlPlaneDefaults{}: {
-					handlers.TenantControlPlaneDefaults{DefaultDatastore: datastore},
+					handlers.TenantControlPlaneDefaults{
+						DefaultDatastore:      datastore,
+						DefaultServiceAccount: serviceAccount,
```

Comment:
No need for this, it can be achieved with the `kubebuilder` marker, see previous comment.

Reply:
I believe I made a [follow up comment about this](https://github.com/clastix/kamaji/discussions/434#discussioncomment-9152589) and you didn't comment on it afterwards so I supposed you were ok with it, but apparently not. 
The reason behind this was to be able to use a specific service account to be considered as "default" by Kamaji, and that can be different from the actual `default` service account. This can be handy when we use the management cluster for something other than just running Kamaji at the same time.
 This will have no effects on the users that do not need this since it will be set to `"default"` by default.

Reply:
I get your point, and sorry if I missed the discussion: too many things happening at the same time.

I would avoid this kind of flag, and rely on configuration over convention: it means that if you need to configure a specific `ServiceAccount` other tools can be used to enforce this behaviour, such as Kyverno policies, or `ClusterClass` in terms of Cluster API.